SCloudRandomizer/MainViewController.m

```diff
@@ -196,16 +197,15 @@ - (IBAction)showTrackInfo:(id)sender {
 #pragma mark - Public/Private methods
 
 - (void)hideAllDataAndControls {
-    [self.btnPlay setImage:[UIImage imageNamed:@"play_btn.png"] forState:UIControlStateNormal];
+    [self.btnPlay setImage:[UIImage imageNamed:@"play_bttn.png"] forState:UIControlStateNormal];
```

Comment:
Got lazy lol. But going forward if we need to change it then we will just keep it as it is.